Paper.tex

```diff
@@ -985,22 +985,23 @@ \section{Block Finalisation} \label{ch:finalisation}
 
 \subsection{Ommer Validation}
 
-The validation of ommer headers means nothing more than verifying that each ommer header is both a valid header and satisfies the relation of $N$th-generation ommer to the present block where $N \leq 6$. The maximum of ommer headers is two. Formally:
+The validation of ommer headers means nothing more than verifying that each ommer header is both a valid header and satisfies the relation of $N$th-generation ommer to the present block where $N \leq 6$. The maximum of ommer headers is two. An ommer may be included only in one block in a blockchain. Formally:
 \begin{equation}
 \lVert B_\mathbf{U} \rVert \leqslant 2 \bigwedge_{U \in B_\mathbf{U}} V(U) \; \wedge \; k(U, P(B_H)_H, 6)
 \end{equation}
 
 where $k$ denotes the ``is-kin'' property:
 \begin{equation}
 k(U, H, n) \equiv \begin{cases} false & \text{if} \quad n = 0 \\
-s(U, H) &\\
-\quad \vee \; k(U, P(H)_H, n - 1) & \text{otherwise}
+(s(U, H) &\\
+\quad \vee \; k(U, P(H)_H, n - 1)) &\\
+\quad \wedge \; U \notin B(H)_\mathbf{U} & \text{otherwise}
 \end{cases}
 \end{equation}
 
 and $s$ denotes the ``is-sibling'' property:
 \begin{equation}
-s(U, H) \equiv (P(H) = P(U)\; \wedge \; H \neq U \; \wedge \; U \notin B(H)_\mathbf{U})
+s(U, H) \equiv (P(H) = P(U)\; \wedge \; H \neq U)
```

Comment:
In order to be a sibling you need to rule out that they are not in the same height in the chain, thus AIUI why we have `\; \wedge \; U \notin B(H)_\mathbf{U})` here. Moving it to the is-kin property doesn't make sense.